interop-reactiveStreams/jvm/src/test/scala/scalaz/zio/interop/reactiveStreams/SubscriberToSinkTest.scala

```diff
@@ -0,0 +1,23 @@
+package scalaz.zio.interop.reactiveStreams
+
+import org.scalatest.{ Matchers, WordSpec }
+import scalaz.zio.DefaultRuntime
+import scalaz.zio.stream.{ Sink, Stream }
+
+class SubscriberToSinkTest extends WordSpec with Matchers with DefaultRuntime {
+
+  "A SubscriberSink" should {
+    "work" in {
+      val seq = Seq.range(0, 100)
+      val resultZIO =
+        for {
+          subRIO            <- Sink.collect[Int].toSubscriber()
+          (subscriber, rio) = subRIO
+          sink              <- subscriber.toSink
```

Comment:
Do you think it's worth testing these conversions by using a Sink written in Akka Streams and converting it to ZIO?

Reply:
done